vnext/ReactWindowsCore/IWebSocket.h

```diff
@@ -73,41 +81,115 @@ struct IWebSocket
 
   #pragma endregion // Inner types
 
+  /// <summary>
+  /// Creates an <c>IWebSocket</c> instance.
+  /// </summary>
+  /// <param name="url">
+  /// WebSocket URL address the instance will connect to.
+  /// The address's scheme can be either ws:// or wss://.
+  /// </param>
   static std::unique_ptr<IWebSocket> Make(const std::string& url);
```

Comment:
>Make [](start = 37, length = 4)

Do we have a UWP implementation ?
Also, I'd love to see us making it explicit that this is a Websocket client, not server ...

Reply:
I will refactor this file and types as `IWebSocketResource` in a subsequent change.
For now, I will add comments to make the client nature of this type clear.

Reply:
@mganandraj Yes, there is a UWP implementation, but the code base is maintained by @ahimberg and his team, in the `ReactUWP` project.

Implementations are not at all related.